src/applications/find-forms/components/SearchResult.jsx

```diff
@@ -1,8 +1,28 @@
+// Node modules.
 import React from 'react';
 import moment from 'moment';
-
+// Relative imports.
 import * as customPropTypes from '../prop-types';
 
+const onDownloadLink = (form) => (event) => {
+  // Derive if the download is a PDF.
+  const isPDF = form?.attributes?.url.toLowerCase().includes('.pdf');
+
+  // IE10+ : (has Blob, but not a[download] or URL)
+  if (navigator.msSaveBlob && isPDF) {
+    event.preventDefault();
+
+    // Make the request to download the PDF.
+    const xhr = new XMLHttpRequest();
+    xhr.open('GET', form?.attributes?.url, true);
```

Comment:
To get to this function in the program execution, the `form` has to have a `url`, so we don't need the optional chaining. We could also just pass the `form.attributes.url` parameter into the function if the isolated string value is easier to work with.

Reply:
We need the form.id value to compose the filename as well 🙂

Reply:
Gotcha, I still think we can do away with the optional chaining. Otherwise it looks like we might send a request to `[object Object]` if `url` is missing

Reply:
Or a request to `false`, or something like that. `?`s are confusing 🤷